backend/app/api/ws/device_namespace.py

```
@@ -0,0 +1,942 @@
+            task_id=task_id,
+            subtask_id=subtask_id,
+            delta_content=delta_content,
+            full_content=full_content,
+            new_offset=new_offset,
+            has_thinking=has_thinking,
+            thinking=thinking,
+            workbench=workbench,
+            is_completed=is_completed,
+        )
+
+    except Exception as e:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)